lib/scope.rb

```diff
@@ -26,9 +26,14 @@ def self.all_scopes
       @scopes
     end
 
-    def self.class_for(scope_name)
+    def self.initialize_scope(scope_name, json, scope_file_store)
```

Comment:
Those are a little bit different, because they only return the appropriate class, not an instance like here.